TrackCat/templatetags/navigation.py

```diff
@@ -0,0 +1,25 @@
+from django import template
+from django.core import urlresolvers
+ 
+register = template.Library()
+ 
+@register.simple_tag(takes_context=True)
+def current(context, url_name, return_value=' active', **kwargs):
+    matches = current_url_equals(context, url_name, **kwargs)
+    return return_value if matches else ''
+ 
```

Comment:
and this one... you don't need to delete the whole line, just the space characted at the start of it...
(the same was true about the other empty lines as well)